docs/ossm/ambient/observability.md

```diff
@@ -0,0 +1,366 @@
+
+### Procedure
+
+1. Navigate to the Red Hat build of OpenTelemetry Operator and install the `OpenTelemetryCollector` custom resource in the `istio-system` namespace. For example,
+
+```sh
+cat <<EOF | kubectl apply -f-
+kind: OpenTelemetryCollector
+apiVersion: opentelemetry.io/v1beta1
+metadata:
+  name: otel
+  namespace: istio-system
+spec:
+  mode: deployment
+  observability:
+    metrics: {}
+  deploymentUpdateStrategy: {}
+  config:
+    exporters:
+      otlp:
+        endpoint: 'tempo-sample-distributor.tempo.svc.cluster.local:4317'
+        tls:
+          insecure: true
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: '0.0.0.0:4317'
+          http: {}
+    service:
+      pipelines:
+        traces:
+          exporters:
+            - otlp
+          receivers:
+            - otlp
+EOF
+```
+
+  NOTE: The `exporters.otlp.endpoint` field is the Tempo sample distributor service in a namespace such as `tempo`.
+
+2. Configure Red Hat OpenShift Service Mesh `Istio` custom resource to define a tracing provider in the spec.values.meshConfig:
+
+```yaml
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+#  ...
+  name: default
+spec:
+  namespace: istio-system
+  profile: ambient
+#  ...
+  values:
+    meshConfig:
+      enableTracing: true
+      extensionProviders:
+      - name: otel
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local
+    pilot:
+      trustedZtunnelNamespace: ztunnel
+```
+
+  NOTE: The `service` field is the OpenTelemetry collector service in the `istio-system` namespace.
+
+3. Create an Istio Telemetry custom resource to enable the tracing provider defined in the spec.values.meshConfig.ExtensionProviders:
+
+```sh
+cat <<EOF | kubectl apply -f-
+apiVersion: telemetry.istio.io/v1
+kind: Telemetry
+metadata:
+  name: otel-demo
+  namespace: istio-system
+spec:
+  tracing:
+    - providers:
+        - name: otel
+      randomSamplingPercentage: 100
+EOF
+```
+
+  NOTE: Once you verify that you can see traces, lower the randomSamplingPercentage value or set it to default to reduce the number of requests.
+
+  NOTE: You may use a spec.targetRefs field to enable tracing at a gateway or a waypoint level.
+
+  NOTE: When you need to use a single Istio Telemetry custom resource for a metrics `prometheus` provider and a tracing provider, you can set the `spec.metrics.overrides.disabled: false` to enable a metrics `prometheus` provider. This step is not needed if you follow the `Configuring Metrics with OpenShift Cluster Monitoring` approach above.
+
+### Tracing Validation
+
+1. Create a `bookinfo` namespace and enable ambient mode by running the following commands:
```

Comment:
Unless there are differences, I don't think you need to repeat the Bookinfo install steps. For the metrics section above, you just had:

> 1. The example below will use the sample Bookinfo application. If you have not already done so, deploy the sample Bookinfo applications. The steps can be found [here](https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ambient/README.md#36-about-the-bookinfo-application).
And it looks like that would work here too. 

Then step 4 could be "4. **If you have not already done so**, deploy a waypoint proxy and enroll the `bookinfo` namespace to use the waypoint: "